docs/diagrams/onchain/permissionedCalls.md

```diff
@@ -29,8 +29,6 @@ sequenceDiagram
         A->>C: permissionedCall
         Note over C,A: send intended calldata wrapped with special selector
     end
-    opt
-        A->>P: assertSpend
-        Note over A,P: assert spend within rolling limit
-    end
+    A->>P: assertSpend
+    Note over A,P: assert spend within rolling limit
```

Comment:
diagram change intentional, the `assertSpend` call is no longer optional because we need to check if the paymaster being used should add gas spend for accounting. We could technically omit the `assertSpend` call if we know the paymaster address is zero, but I feel like this complicates the code flow enough where I'd prefer to keep the logic more clearly separated.